_includes/layouts/blog.html

```diff
@@ -40,18 +40,16 @@ <h1> Blog </h1>
                     margin-top-4 padding-right-4 
                     usa-prose"
     >
-
+<!--
       <div> 
         <h3 class="heading--sm margin-top-0 margin-bottom-1">
           About the .gov blog
         </h3>
         <p> This blog is super-cool and you should read it </p>
       </div>
+-->
```

Comment:
Would you mind just deleting this rather than commenting it out? That way the comment won't show up in the HTML either. We can always put it back in.